### DISCUSSION TITLE
How can I make a tilemap with 2 layers and only use array data?

### DISCUSSION THREAD

**Discussion by chenxxzhe** [photonstorm/phaser#6315] [Q&A]
for example
```js
const map = this.make.tilemap({
  key: 'map',
  data: [[1,2,], [1,2]],  // how do I define two layer here ?
})
const tiles  = map.addTilesetImage('tiles')
const layer = map.createLayer(0, tiles)  // and why can I only use 0 to show the layer ? I try other name, and the layer does not render.
```

**Comment by pacifiquem** [comment]
basicly, creating two maps with the same dimensions, set the z-order with setDepth, and if the map above has transparent tiles, or tiles with the ID -1, the map below should be visible.

I personally never used it for larger maps, but I assume it should not cause performance issue, and is an easy solution.

**Reply by pacifiquem** [reply]
Or use this [link to explained solution](https://stackoverflow.com/questions/74255201/tilemap-depth-sorting-by-tile-id/74256267#74256267)
